pontoon/sync/templates/sync/log_list.html

```diff
@@ -47,13 +47,13 @@ <h4>
                 {% endif %}
                 </h4>
               </td>
-              <td class="last-status {{ format_sync_status_class(project.sync_status) }}">
+              <td class="latest-status {{ format_sync_status_class(project.sync_status) }}">
                 {{ format_sync_status_label(project.sync_status) }}
               </td>
-              <td class="last-start">
-                {{ project.sync_start_time|date("Y-m-d H:i:s") or "―" }}
+              <td class="latest-start">
+                <time {% if project.sync_start_time %}datetime="{{ project.sync_start_time.isoformat() }}" title='{{ project.sync_start_time|date("Y-m-d H:i:s") }} UTC'{% endif %}>{{ project.sync_start_time|naturaltime or "―" }}</time>
               </td>
-              <td class="duration" title='{% if project.sync_end_time %}Last end: {{ project.sync_end_time|date("Y-m-d H:i:s") }}{% endif %}' data-sort="{{ format_sync_duration(project.sync_start_time, project.sync_end_time, ms=True) }}">
+              <td class="duration" title='{% if project.sync_end_time %}Latest End: {{ project.sync_end_time|date("Y-m-d H:i:s") }} UTC{% endif %}' data-sort="{{ format_sync_duration(project.sync_start_time, project.sync_end_time, ms=True) }}">
```

Comment:
The start time formatting (here and also in the other files) uses `.isoformat()`, but that's not included for end times. Should it be?

Reply:
`.isoformat()` is only used in the `datetime` attribute, which is used for sorting.